fastly/resource_fastly_service_v1.go

```diff
@@ -1452,6 +1452,13 @@ func resourceServiceV1() *schema.Resource {
 							Computed:    true,
 							Description: "Generated dictionary ID",
 						},
+						"write_only": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Default:     false,
+							ForceNew:    true,
```

Comment:
`ForceNew` indicates that any change in this field requires the resource to be destroyed and recreated. It makes totally sense to me to apply it here but it doesn't work because of the implementation of the Fastly provider. The reason is that a dictionary is not a standalone resource! Changing this field results to:

```
Terraform will perform the following actions:

  # fastly_service_v1.test must be replaced
-/+ resource "fastly_service_v1" "test" {
        activate       = true
      ~ active_version = 1 -> (known after apply)
        comment        = "Managed by Terraform"
      + default_host   = (known after apply)
        default_ttl    = 3600
      + force_destroy  = true
      ~ id             = "609pjtHtRLoZkvK5IuGnI1" -> (known after apply)
        name           = "demofastly"

        backend {
            address               = "1.2.3.4"
            auto_loadbalance      = true
            between_bytes_timeout = 10000
            connect_timeout       = 1000
            error_threshold       = 0
            first_byte_timeout    = 15000
            max_conn              = 200
            name                  = "localhost"
            port                  = 80
            ssl_check_cert        = true
            use_ssl               = false
            weight                = 100
        }

      - dictionary { # forces replacement
          - dictionary_id = "1gaW92CKUj3mYKn6iY9bvF" -> null
          - name          = "my_dictionary" -> null
          - write_only    = true -> null
        }
      + dictionary { # forces replacement
          + dictionary_id = (known after apply)
          + name          = "my_dictionary"
          + write_only    = false
        }

        domain {
            name = "www.test.com"
        }
    }

Plan: 1 to add, 0 to change, 1 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

fastly_service_v1.myservice: Destroying... [id=609pjtHtRLoZkvK5IuGnI1]

Error: 400 - Bad Request:

    Title:  Bad request
    Detail: Cannot delete service with active version
```

Reply:
bummer... I'd hoped that it was just the sub-resource that would be destroyed